rules/no-identical-title.js

```diff
@@ -39,7 +38,7 @@ module.exports = function (context) {
 			var args = node.arguments;
 			var titleNode = args.length > 1 || ava.hasTestModifier('todo') ? args[0] : undefined;
 
-			if (!isStatic(titleNode)) {
+			if (titleNode === undefined || !isStatic(titleNode)) {
```

Comment:
Checking explicitly here so we don't treat empty strings or whatever as anonymous tests.